circle.yml

```diff
@@ -26,7 +26,7 @@ machine:
     DEP3: " && /home/ubuntu/checkstyle/groovy-2.4.7/bin/groovy launch.groovy --listOfProjects projects-to-test-on.properties --config my_check.xml"
     TESTER_DEPENDENCIES: $CMD1$DEP1$DEP2$DEP3
     CMD2: " && sed -i'' 's/^openjdk/#openjdk/' projects-for-circle.properties"
-    CMD3: " && /home/ubuntu/checkstyle/groovy-2.4.7/bin/groovy launch.groovy --listOfProjects projects-for-circle.properties --config checks-nonjavadoc-error.xml"
+    CMD3: " && cd ../.. /home/ubuntu/checkstyle/groovy-2.4.7/bin/groovy contribution/checkstyle-tester/launch.groovy --listOfProjects projects-for-circle.properties --config checks-nonjavadoc-error.xml --checkstyleVersion ${CS_POM_VERSION}"
```

Comment:
Please explain reason of this "cd" .
the less "cd" is used in shell script, the better.

Reply:
please also put `&&` between `cd` command ` groovy` command so it is clear where the `cd` command stops and and the `groovy` command begins. I mistakenly thought you were cding into a groovy directory.

Reply:
@rnveach done
@romani I was only able to make it work this way. Without `cd` I was getting 
> Goal requires a project to execute but there is no POM in this directory (/home/ubuntu).

like here: https://circleci.com/gh/Vladlis/checkstyle/324

Reply:
well, CI was green only due to a missed `cd` and now this doesn't work either